subprojects/parseq/src/test/java/com/linkedin/parseq/ParSeqUnitTestHelperTest.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.parseq;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertThrows;
+
+public class ParSeqUnitTestHelperTest {
```

Comment:
What is this class used for?

Reply:
It's a unit test for "ParSeqUnitTestHelper" class